x/pairing/keeper/pairing.go

```diff
@@ -152,15 +154,29 @@ func (k Keeper) getPairingForClient(ctx sdk.Context, chainID string, clientAddre
 		return nil, 0, false, fmt.Errorf("did not find providers for pairing: epoch:%d, chainID: %s", block, chainID)
 	}
 
-	providers, err = k.calculatePairingForClient(ctx, possibleProviders, projectToPair, block, chainID, geolocation, epochHash, providersToPair)
+	selectedProvidersFlag := false
+	switch selectedProvidersMode {
```

Comment:
use if case, there is only 1 option

Reply:
but there will be more in the future (mix)